scipy/sparse/csgraph/tests/test_shortest_path.py

```diff
@@ -419,9 +449,11 @@ def test_yen_undirected():
         source=0,
         sink=3,
         K=4,
+        directed=False,
```

Comment:
What's going on here? Is a bug being fixed/behavior being changed?

Reply:
bug fixed. The test is meant to be "undirected" but the parameter was not passed properly.